tests/taskcluster/bin/raptor_graph

```diff
@@ -0,0 +1,135 @@
+#! /usr/bin/env node
+
+/**
+Entrypoint for the task graph extension / decisions for gaia raptor tests.
+*/
+
+var decorate = require('../lib/decorate_task');
+var yaml = require('js-yaml');
+var fs = require('fs');
+var fsPath = require('path');
+var slugid = require('slugid');
+var util = require('util');
+var template = require('json-templater/object');
+
+// Cache for task definitions...
+var TASK_DEFINITIONS = {};
+function loadTask(name, parameters) {
+  if (!TASK_DEFINITIONS[name]) {
+    var path = fsPath.resolve(__dirname + '/../tasks/' + name + '.yml');
+    TASK_DEFINITIONS[name] = yaml.safeLoad(fs.readFileSync(path));
+  }
+  return decorate(TASK_DEFINITIONS[name], parameters);
+}
+
+function params() {
+  return Array.prototype.slice.call(arguments).reduce(function(target, props) {
+    for (var key in props) target[key] = props[key];
+    return target;
+  }, {});
+}
+
+// Map of task ids to task names.
+var taskIdMap = {};
+
+var config = {
+  params: {
+    provisionerId: 'aws-provisioner',
+    workerType: 'emulator-ics',
+    // TO-DO: Once builds are indexed, query TC to get taskID & get URL for latest build
+    emulatorURL: 'https://queue.taskcluster.net/v1/task/_CZfx9s8T6C14mZG9Eo9gA/artifacts/public/build/emulator.tar.gz',
```

Comment:
So James added indexes for builds now, but they might be changing to be under a "buildbot" namespace.  Also, the index right now just returns the task ID with the highest rank related to the particular index you're searching and not specific artifacts for that task.  I think jonas was mentioning something where you could specific an artifact name and it will translate that into the artifact URL for the task matching the index you're retrieving.

https://tools.taskcluster.net/index/#gecko.v1.b2g-inbound.latest.emulator-ics/gecko.v1.b2g-inbound.latest.emulator-ics.opt